generator/lib/src/generator.dart

```diff
@@ -362,7 +362,7 @@ class RetrofitGenerator extends GeneratorForAnnotation<retrofit.RestApi> {
     String? definePath = method.peek("path")?.stringValue;
     paths.forEach((k, v) {
       final value = v.peek("value")?.stringValue ?? k.displayName;
-      definePath = definePath?.replaceFirst("{$value}", "\${${k.displayName}}");
```

Comment:
Could you please explain me this change? :slightly_smiling_face:

Reply:
@ilovelinux I've added an enum-based value to the `PathTest` below to verify.

Without this change, generator is producing a `.toString()` value for enums used in the path (something like EnumClass.ordinalName) instead of just enum name.